### DISCUSSION TITLE
@ant-design/plots with react 16.9.0

### DISCUSSION THREAD

**Discussion by sanand404** [ant-design/ant-design-charts#2023] [General]
Hello all, I'm trying to use @antd-design/plots:v1.0.8 in React v16.9.0.
`
[09:07:02] [SyntaxError: 'import' and 'export' may appear only with 'sourceType: module'] {
  line: 10,
  column: 1,
  annotated: '\n' +
    '/mnt/d/node_modules/lodash-es/lodash.js:10\n' +
    "export { default as add } from './add.js';\n" +
    '^\n' +
    "ParseError: 'import' and 'export' may appear only with 'sourceType: module'",
  stream: Labeled {
    _readableState: ReadableState {
      objectMode: true,
      highWaterMark: 16,
      buffer: BufferList { length: 0 },
      length: 0,
      pipes: [Labeled],
      pipesCount: 1,
      flowing: true,
      ended: false,
      endEmitted: false,
      reading: true,
      sync: false,
      needReadable: true,
      emittedReadable: false,
      readableListening: false,
      resumeScheduled: false,
      destroyed: false,
      defaultEncoding: 'utf8',
      awaitDrain: 0,
      readingMore: false,
      decoder: null,
      encoding: null
    },
    readable: true,
    _events: [Object: null prototype] {
      end: [Array],
      finish: [Function],
      error: [Function (anonymous)],
      data: [Function: ondata],
      _mutate: [Function]
    },
    _eventsCount: 5,
    _maxListeners: undefined,
    _writableState: WritableState {
      objectMode: true,
      highWaterMark: 16,
      finalCalled: false,
      needDrain: false,
      ending: false,
      ended: false,
      finished: false,
      destroyed: false,
      decodeStrings: true,
      defaultEncoding: 'utf8',
      length: 1,
      writing: true,
      corked: 0,
      sync: true,
      bufferProcessing: false,
      onwrite: [Function (anonymous)],
      writecb: [Function: nop],
      writelen: 1,
      bufferedRequest: null,
      lastBufferedRequest: null,
      pendingcb: 1,
      prefinished: false,
      errorEmitted: false,
      bufferedRequestCount: 0,
      corkedRequestsFree: [CorkedRequest]
    },
    writable: true,
    allowHalfOpen: true,
    _options: { objectMode: true },
    _wrapOptions: { objectMode: true },
    _streams: [ [DestroyableTransform] ],
    length: 1,
    label: 'syntax',
    [Symbol(kCapture)]: false
  }
}
[09:07:02]
/mnt/d/node_modules/lodash-es/lodash.js:10
export { default as add } from './add.js';
^
ParseError: 'import' and 'export' may appear only with 'sourceType: module'
`

What possible mistake here?

**Comment by lxfu1** [comment]
It seems to be an issue with the tslib version. Try upgrading to the `^2.6.1 version` and try again.
